src/core/isEmpty.spec.js

```diff
@@ -23,6 +28,7 @@ test('isEmpty predicate function', t => {
   t.equal(isEmpty(''), true, 'returns true with empty string')
   t.equal(isEmpty('string'), false, 'returns false with non-empty string')
 
+  t.equal(isEmpty(Fake.empty()), true, 'returns true with empty monoid')
```

Comment:
Might want to add `returns false for non-empty monoids` for consistency. I think that's what the CI is crying about as well?